data/sql/updates/pending_db_world/rev_1473152344430838100.sql

```diff
@@ -0,0 +1,28 @@
+INSERT INTO version_db_world(`sql_rev`) VALUES ('1473152344430838100');
+
+/* 
+I take no credit for this fix im just doing the PR to close the issue.
+Winterfin oricle will now cast lightning bolt.
+the real credit goes to Chrisjiajun, this works
+
+Closes issue #119
```

Comment:
can you please move these texts to the PR description instead of the file itself?

thanks